src/@next/Tabs/TabStyle.ts

```diff
@@ -63,7 +90,7 @@ export const StyledSpan = styled.span`
 
 export const StyledTabButton = styled.button`
```

Comment:
@ninariccimarie The tab now has a blue-ish background color when clicked. Can we remove it?